tools/vscode/refresh_compdb.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+CURRENT_SCRIPT_DIR="$(realpath "$(dirname "${BASH_SOURCE[0]}")")"
+
+# shellcheck source=ci/build_setup.sh
+. "${CURRENT_SCRIPT_DIR}"/../../ci/build_setup.sh
```

Comment:
can we make this optional - ie add a case in do_ci.sh for `refresh_compdb` and then not add here ?

Reply:
also SGTM. I can update it tomorrow.